### DISCUSSION TITLE
are you familiar with UPBGE project?

### DISCUSSION THREAD

**Discussion by BluePrintRandom** [katharostech/arsenal#114] [Questions]
maybe you could join forces with the UPBGE developers?

[a comment by an automated account is not shown]

**Comment by zicklag** [comment]
Hi @BluePrintRandom, we are familiar with UPBGE, I did a little bit of looking into it a while back. Unfortunately UPBGE has a drastically different architecture than Arsenal being that it is a fork of the Blender game engine. This has issues for us because you are not allowed to use Blender code for commercial purposes due to the GPL license. Also, we would really like to make it so that Arsenal is a Blender _plugin_ instead of a fork, so that you don't have to download a different forked version of Blender to use it.

In short, I think there is very little, if anything, that we can probably share with UPBGE in Arsenal because the approaches to the problem are so drastically different and because the GPL code not being something that we can integrate into Arsenal.

---

P.S. the open collective bot is a little confused! I'll fix that. :slightly_smiling_face:
